web-extensions/chrome/engine/datamodel.js

```diff
@@ -258,13 +258,6 @@ BmcComic.prototype.serialize = function() {
     };
 };
 
-/**
```

Comment:
Why remove this piece of documentation ? :o

Reply:
I didn't understand what it was documenting...

Reply:
I'ts documenting a "Callback Type"; which is used in the following documentation block (`{BmcComic~iterCb}`)
This is a good way to document the expected prototypes of the callbacks provided to a function, in JSDoc.